Required for building with CMake 3.4.0.

Submitted via email directly to the upstream author.

commit 2624465f87d7782527ee82567fd3a4fdf4c54241
Author: Raphael Kubo da Costa <rakuco@FreeBSD.org>
Date:   Thu Nov 26 11:58:11 2015 +0100

    cmake: Explicitly include the CheckIncludeFiles module.

    This fixes the build with CMake 3.4.0: kdbg/CMakeLists.txt was previously
    relying on the module being included indirectly.

--- kdbg/CMakeLists.txt
+++ kdbg/CMakeLists.txt
@@ -4,6 +4,7 @@ add_subdirectory(typetables)
 
 include_directories(${KDE4_INCLUDES} ${KDE4_INCLUDE_DIR} ${QT_INCLUDES} )
 include(CheckFunctionExists)
+include(CheckIncludeFiles)
 
 CHECK_INCLUDE_FILES(pty.h HAVE_PTY_H)
 CHECK_INCLUDE_FILES(libutil.h HAVE_LIBUTIL_H)
